code/modules/projectiles/pins.dm

```diff
@@ -305,13 +305,13 @@ var/list/wireless_firing_pins = list() //A list of all initialized wireless firi
 			var/obj/item/gun/energy/EG = gun
 			if(EG.required_firemode_auth[EG.sel_mode] == WIRELESS_PIN_STUN)
 				return TRUE
-			else if (GLOB.security_level == SEC_LEVEL_YELLOW || GLOB.security_level == SEC_LEVEL_RED)
+			else if (GLOB.security_level == SEC_LEVEL_YELLOW || GLOB.security_level == SEC_LEVEL_RED || GLOB.security_level == SEC_LEVEL_BLUE)
```

Comment:
This is becoming unreadable, reformat them as

```suggestion
			else if (GLOB.security_level in list(SEC_LEVEL_YELLOW, SEC_LEVEL_RED, SEC_LEVEL_BLUE))
```

Or even better

```suggestion
			else if (GLOB.security_level == SEC_LEVEL_YELLOW || GLOB.security_level >= SEC_LEVEL_BLUE)
```

This is because you probably want them to remain unlocked at delta alert too, the one time a decade we use it